## Driver Parameters

#publisher rate
rate: 100
#robot base frame
base_frame: base_link
#map frame
root_frame: map
robot_state_topic: /robot_state
vel_state_topic: /gazebo/link_states